src/main/webapp/app/shared/statistics-graph/statistics-average-score-graph.component.scss

```diff
@@ -40,3 +44,14 @@
 .best-color {
     background: var(--stat-av-sc-legend-best);
 }
+
+.chart-container {
+    width: 83.33%;
```

Comment:
I suggest using non-decimal percentages as I am not sure how these are handled by the browser :thinking: